gui/wxpython/main_window/frame.py

```diff
@@ -2278,20 +2278,6 @@ def OnDeleteLayer(self, event):
 
     def OnKeyDown(self, event):
         """Key pressed"""
```

Comment:
There is no reason to leave here this method, I would just remove it and the associated binding.

Reply:
Yes of course.